editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper.spec.browser2.tsx

```diff
@@ -333,6 +333,24 @@ describe('Spy Wrapper Template Path Tests', () => {
         "storyboard/scene-1/app:other-app-root/other-inner-div/other-card-instance": Object {
           "name": "div",
         },
+        "storyboard/scene-1/app:other-app-root/other-inner-div/other-card-instance:other-button-instance/other-expr/other-hi-element~~~1:c33/hi-element-fragment-child-1": Object {
```

Comment:
Are these test changes a leftover from the previous approach?

Reply:
Nope, we collect more than we used to.

Reply:
we definitely do not want new metadata entries introduced by this PR.